include/picongpu/fields/incidentField/Functors.hpp

```diff
@@ -549,7 +549,7 @@ namespace picongpu
                         float3_X internalPosition = this->getInternalCoordinates(totalCellIdx);
                         internalPosition[0] = 0.0_X;
                         auto const w0 = float3_X(1.0_X, Unitless::W0_AXIS_1, Unitless::W0_AXIS_2);
-                        auto const r2 = pmacc::math::abs2(internalPosition / w0);
+                        auto const r2 = pmacc::math::l2norm2(internalPosition / w0);

```

Comment:
not needed here (since it was originally already like this)  but it would be computationally more efficient to move the division outside the norm:
```suggestion
                        auto const r2 = pmacc::math::l2norm2(internalPosition) / w0;

```

Reply:
discussed offline: This will not be equivalent.